src/Microsoft.DotNet.ProjectModel/FileFormatException.cs

```diff
@@ -29,20 +31,11 @@ public override string ToString()
 
         internal static FileFormatException Create(Exception exception, string filePath)
         {
-            if (exception is JsonDeserializerException)
```

Comment:
This looks like you'll lose some information. I believe there's an equivalent thing you can cast to for the JSON.NET specific exception.